newsletter-january-2022.rst

```diff
@@ -0,0 +1,88 @@
+.. post:: January 12, 2022
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in January.
+
+Read the Docs newsletter - January 2022
+=======================================
+
+Welcome to the latest edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
+
+Company highlights
+------------------
+
+- We are now managing custom domains for our corporate users using Cloudflare SSL for SaaS,
+  which will remove the manual work that was needed on our side
+  and make the process of setting up a custom domain almost instantaneous.
+
+New features
```

Comment:
Here, we could say that the analytics cookie became a session cookie: https://github.com/readthedocs/readthedocs.org/pull/8694

Reply:
Done!